pos_folio/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# See LICENSE file for full copyright and licensing details.
```

Comment:
This line in all py to be replaced by:
'# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).'